### DISCUSSION TITLE
react-pivottable

### DISCUSSION THREAD

**Discussion by Lonli-Lokli** [DefinitelyTyped/DefinitelyTyped#53954] [Request a new @types package] [answered]
https://github.com/plotly/react-pivottable

react-pivottable is a React-based pivot table library with drag'n'drop functionality.

**Comment by hozblok** [comment, marked as answer]
Hi @Lonli-Lokli ,

I assume it was done here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60653
